src/SlashCommands.js

```diff
@@ -208,6 +210,47 @@ export const CommandMap = {
         },
     }),
 
+    roomavatar: new Command({
+        name: 'roomavatar',
```

Comment:
This sounds like it'll modify the avatar of the room rather than your own. Although the description clarifies what happens, lots of people don't read that. Can we make this `myroomavatar` or something instead?

Reply:
Should roomnick also become myroomnick then?

Reply:
I think roomnick is fine, because rooms don't have nicknames. Rooms do have avatars though. Maybe we advertise it as `myroomnick` and support `roomnick` as an alias (to avoid breaking existing third party documentation)?